Sources/AppcuesKit/Data/Models/Experience.swift

```diff
@@ -174,9 +174,10 @@ extension Experience: Decodable {
         redirectURL = try? container.decode(URL.self, forKey: .redirectURL)
         nextContentID = try? container.decode(String.self, forKey: .nextContentID)
 
-        // Check for an experience-level embed trait
+        // Check for an experience-level or first group-level embed trait
+        let eligibleTraits = traits + (steps.first?.traits ?? [])
         if #available(iOS 13.0, *),
-           let embedTrait = self.traits.first(where: { $0.type == AppcuesEmbeddedTrait.type }),
+           let embedTrait = eligibleTraits.first(where: { $0.type == AppcuesEmbeddedTrait.type }),
```

Comment:
@andretortolano fyi on this change, from [thread with builder team ](https://appcues.slack.com/archives/C031RLGS4F8/p1691089870467149)